common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -255,7 +255,7 @@ def test_fields_on_my_private_profile(self):
         Scenario: Verify that desired fields are shown when looking at her own private profile.
 
         Given that I am a registered user.
-        And I visit My Profile page.
+        And I visit my Profile page.
```

Comment:
This was the "my" that I thought might be what was breaking tests. (without actually seeing the test output, so who knows.)